markdown/extensions/wikilinks.py

```diff
@@ -51,7 +51,7 @@ def extendMarkdown(self, md):
         self.md = md
 
         # append to end of inline patterns
-        WIKILINK_RE = r'\[\[([\w0-9_ -]+)\]\]'
+        WIKILINK_RE = r'\[\[([\w0-9_ -\.]+)\]\]'
```

Comment:
Regardless of whether this is accepted or not, `-` should be at the end so it is not interpreted as specifying a range of characters between space and dot.

Reply:
That's a good suggestion, thanks.